learn/guides/organize-ballerina-code/package-references.md

```diff
@@ -105,6 +112,12 @@ Ballerina strictly follows the rules of [Semantic Versioning](https://semver.org
 
 *   Once the package is production-ready, you can use a stable version (E.g. 1.0.0). Any subsequent minor or patch releases of the same major version should be backward compatible and, should not break existing builds.
 
+### The `include` field
+
+You can provide paths to any additional resources that needs to be packed in the BALA file during the use of `bal pack` command.
+
+The include field is an array that accepts relative paths from the root of the package to the files or directories. 
```

Comment:
The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory.